recommender/static/html/recommender.html

```diff
@@ -76,18 +78,15 @@
 <div class="recommenderBlock">
     <div class="recommenderRecommendations">
         <div class="recommenderContent">
-            <div class="recommender_hideShow" role='button' tabindex='0' data-step="1" aria-expanded='false' data-intro="This is a list of recommended resources. If you're stuck, curious, or want to learn more, please look around. If you find a useful resources, help your fellow classmates by adding it!">
-                Show related resources
+            <div class="recommender_hideShow" role='button' tabindex='0' data-step="1" aria-expanded='false' data-intro="${_('This is a list of recommended resources. If you are stuck, curious, or want to learn more, please look around. If you find a useful resources, help your fellow classmates by adding it!')}">
```

Comment:
I don't want to assume too much, but is the switch from `you're` to `you are` just because `gettext` is using single quoted strings?

If so, this should probably be
```suggestion
            <div class="recommender_hideShow" role='button' tabindex='0' data-step="1" aria-expanded='false' data-intro="${_('This is a list of recommended resources. If you\'re stuck, curious, or want to learn more, please look around. If you find a useful resources, help your fellow classmates by adding it!')}">
```